cpp/ql/lib/change-notes/2024-10-04-models-as-data.md

```diff
@@ -0,0 +1,4 @@
+---
+category: feature
+---
+* Models-as-Data support has been added for C/C++. This feature allows flow sources, sinks and summaries to be expressed in compact strings as an alternative to modelling each source / sink / summary with explicit CodeQL. See `dataflow/ExternalFlow.qll` for documentation and specification of the model format, and `models/implementations/ZMQ.qll` for a simple example of models. Importing models from `.yml` is not yet supported.
```

Comment:
QL is the _language_, and CodeQL is the _product_. I think you want to refer to the language here, right?
```suggestion
* Models-as-Data support has been added for C/C++. This feature allows flow sources, sinks and summaries to be expressed in compact strings as an alternative to modelling each source / sink / summary with explicit QL. See `dataflow/ExternalFlow.qll` for documentation and specification of the model format, and `models/implementations/ZMQ.qll` for a simple example of models. Importing models from `.yml` is not yet supported.
```

Reply:
Thanks, I'd forgotten that distinction.